docs/source/examples/Natural and artificial perturbations.ipynb

```diff
@@ -97,7 +106,7 @@
    "source": [
     "plt.ylabel(\"h(t)\")\n",
     "plt.xlabel(\"t, days\")\n",
-    "plt.plot(tofs.value, rr.norm() - Earth.R);"
+    "plt.plot(tofs.value, rr.data.norm() - Earth.R);"
```

Comment:
Oops, I didn't notice these ones. You were probably using the latest stable version, but it changed in the development version.